README.md

```diff
@@ -8,7 +8,22 @@ Among these tasks are:
 * **View|Place|Modify** an order and/or order item (e.g. update an order item, cancel the order)
 * **Tables|Seats Management** - reserve a table, view|modify the tables chart
 * **Reservations Management** - Update|Cancel a reservation/booking, print booking summary, process payment
-### Contributing
+* 
 
-Please read the [contribution guidelines](https://github.com/NextPP/sudoManager/blob/main/CONTRIBUTING.md) before starting to work on a pull request.
 
+## Build
```

Comment:
Lets have these headers:
- Requirements(cmake install with OS package manager, and cmake with pip)
- Build(with commands on how to build, check the [modern cmake guide](https://cliutils.gitlab.io/modern-cmake/chapters/intro/running.html))
- Usage(*bluey* is the way the user interacts with the system)

It would be thrilling to have **Windows** build guide as well(VSCode, or VS). (Optional, if you like to :))

Feel free to get inspired from other repos on Github
or [here](https://github.com/ttroy50/cmake-examples)